packages/legend-query-builder/src/components/result/tds/QueryBuilderTDSSimpleGridResult.tsx

```diff
@@ -47,6 +48,47 @@ import type {
 } from '../../../stores/QueryBuilderResultState.js';
 import { QUERY_BUILDER_TEST_ID } from '../../../__lib__/QueryBuilderTesting.js';
 
+export const getFloatGridColumnCustomHeader = (
+  columnName: string,
+): string => ` <div class="query-builder__result__values__table__custom-header">
+              <div>${columnName}</div>
```

Comment:
i dont see where we actually do the rounding ? 
Shouldnt the CellFormatter need editing here?

Reply:
Currently, in grid, we set ` maximumFractionDigits: 4` see in `QueryResultCellRenderer`

```
    const formattedCellValue = (): QueryBuilderTDSResultCellDataType => {
      if (isNumber(cellValue)) {
        return Intl.NumberFormat(DEFAULT_LOCALE, {
          maximumFractionDigits: 4,
        }).format(Number(cellValue));
      } else if (isBoolean(cellValue)) {
        return String(cellValue);
      }
      return cellValue;
    };
```

This is for displaying a warning icon to remind users of abandoning some fraction digits there are more than 4.